packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -12544,6 +12544,150 @@ void main() {
       );
     });
   });
+
+  group('Spell check', () {
+    testWidgets(
+        'Spell check configured properly when spell check disabled',
+            (WidgetTester tester) async {
+          await tester.pumpWidget(
+            MaterialApp(
+              home: EditableText(
+                controller: TextEditingController(text: 'A'),
+                focusNode: FocusNode(),
+                style: const TextStyle(),
+                cursorColor: Colors.blue,
+                backgroundCursorColor: Colors.grey,
+                cursorOpacityAnimates: true,
+                autofillHints: null,
+              ),
+            ),
+          );
+
+          final EditableTextState state =
+            tester.state<EditableTextState>(find.byType(EditableText));
+          expect(state.spellCheckEnabled, isFalse);
+        });
+
+    testWidgets(
+        'Error thrown when spell check configuration defined without specifying misspelled text style',
```

Comment:
Is there a useful message for the user in this case?

Reply:
Yes! It's in the EditableText [constructor](https://github.com/flutter/flutter/pull/104460/files#diff-f5f93c879cef9a102adbf148583de9f3d9a05f9678fdcad7bde106f62200bb34R705).